### DISCUSSION TITLE
Notice: Off-Campus Bastion Access to WATcloud Disabled – Use VPN Instead

### DISCUSSION THREAD

**Discussion by Jimmyj30** [WATonomous/infrastructure-support#63] [General Announcements]
Hi @WATonomous/watcloud-compute-cluster-users,

On 2024-12-18, the University required WATcloud to move all machines behind the University [firewall](https://cloud.watonomous.ca/docs/compute-cluster/firewall). This means users can **only access WATcloud while connected to the University network** (such as the University VPN). As a result, **off-campus cluster access through the bastion host will no longer work**.

For updated instructions on how to access the cluster through SSH, please refer to [this link](https://cloud.watonomous.ca/docs/compute-cluster/ssh).

Note that this change will come into effect after the maintenance in https://github.com/WATonomous/infrastructure-support/discussions/64 is done.

### TLDR: Connect to the UWaterloo VPN before accessing the cluster normally.

**Comment by ben-z** [comment]
Maintenance #64 is complete. To reiterate, access to the cluster is currently restricted to campus users or those connected to the [university VPN](https://cloud.watonomous.ca/docs/compute-cluster/firewall#vpn). The only change to your development workflow is the need to connect to the VPN or be on campus for SSH to function.

For non-uwaterloo students--we are currently working with the university for a solution. If you currently have work on the cluster, please [contact us](https://cloud.watonomous.ca/docs/compute-cluster/support-resources) and we will make sure that your data is available to you.

As always, you can find the latest access instructions on our website: https://cloud.watonomous.ca/docs/compute-cluster/ssh
